WordPress/Classes/ViewRelated/Blog/QuickStartChecklistHeader.swift

```diff
@@ -60,8 +63,54 @@ class QuickStartChecklistHeader: UIView {
     @IBAction private func headerDidTouch(_ sender: UIButton) {
         collapse.toggle()
     }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        prepareForVoiceOver()
+    }
 }
 
 private enum Constant {
     static let title = NSLocalizedString("Complete (%i)", comment: "The table view header title that displays the number of completed tasks")
 }
+
+// MARK: - Accessible
+
+extension QuickStartChecklistHeader: Accessible {
+    func prepareForVoiceOver() {
+        // Here we explicit configure the subviews, to prepare for the desired composite behavior
+        bottomStroke.isAccessibilityElement = false
+        contentView.isAccessibilityElement = false
+        titleLabel.isAccessibilityElement = false
+        chevronView.isAccessibilityElement = false
+
+        // Neither the top stroke nor the button (overlay) are outlets, so we configured them in the nib
+
+        // From an accessibility perspective, this view is essentially monolithic, so we configure it accordingly
+        isAccessibilityElement = true
+        accessibilityTraits = [ .header, .button ]
```

Comment:
We're not terribly consistent, but both our reference style guides have so spaces inside array list brackets: `[.header, .button]`

Reply:
I'll tidy this up.